package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@connectedcars/logutil",
-  "version": "5.0.3",
+  "version": "6.0.0",
```

Comment:
We usually don't update the version until after the PR has been approved and merged into master (that way we can use the `npm version` command which automatically updates the package files and creates git tag).

Reply:
No, we just push that version bump directly to master. The teams that usually need to bump the versions on each NPM package repo has permission to bypass the branch protection. I've added the device team as well to this repo and there is a step-by-step guide for publishing a new version here: https://wiki.connectedcars.io/backend/packages/npm#updating-subsequent-publishes